components/observations/SimpleForm.tsx

```diff
@@ -116,7 +116,7 @@ export const SimpleForm: React.FC<{
           exif: true,
           mediaTypes: ImagePicker.MediaTypeOptions.Images,
           preferredAssetRepresentationMode: ImagePicker.UIImagePickerPreferredAssetRepresentationMode.Compatible,
-          quality: 0.2,
+          quality: 0.9,
```

Comment:
I made it 0.2 in https://github.com/NWACus/avy/pull/521 when I was trying to eliminate crashes from ImagePicker invocations. I didn't look carefully enough at image quality because I was just uploading test images from the iOS simulator 🙈